mkdocs.yml

```diff
@@ -7,34 +7,36 @@ site_dir: 'site' # default
 nav:
     - Home: index.md
     - Getting Started: pages/GettingStarted.md
-    - Workflows: pages/Workflows.md
```

Comment:
Can/Should we do a redirect for the removed Workflows page?

Reply:
Since the 404 page still shows the navigation tree, I wouldn't bother too much: https://sap.github.io/ui5-tooling/pages/xyz/